src/edge/io.py

```diff
@@ -39,7 +39,12 @@ def to_gff_file(self, file):
             for annotation in fragment.annotations():
                 # FeatureLocation first bp is AfterPosition, so -1
                 loc = FeatureLocation(annotation.base_first - 1, annotation.base_last)
-                qualifiers = {'name': annotation.feature.name}
+                qualifiers = annotation.feature.qualifiers
+                if "phase" in qualifiers and qualifiers["phase"] is None:
+                    qualifiers["phase"] = 0
+                elif "Phase" in qualifiers and qualifiers["Phase"] is None:
```

Comment:
I suppose the input gff could have both versions 'phase' or 'Phase' so we have to deal with both cases?

Reply:
I've seen GFF that use capital first letter, and all lower letters. I don't think there's a set specification.